src/ci/github-actions/ci.yml

```diff
@@ -229,6 +232,19 @@ x--expand-yaml-anchors--remove:
           TOOLSTATE_REPO_ACCESS_TOKEN: ${{ secrets.TOOLSTATE_REPO_ACCESS_TOKEN }}
         <<: *step
 
+      - name: create github artifacts
+        run: src/ci/scripts/create-doc-artifacts.sh
+        <<: *step
+
+      - name: upload artifacts to github
+        uses: actions/upload-artifact@v3
+        with:
+          name: rust-ci-${{ env.SHORT_SHA }}
```

Comment:
I understand wanting this to be generic, but I think it would be better to have dedicated artifacts with meaningful names.

For instance, I think it would be helpful if this were named something like `library-docs-SHA` instead.

Reply:
Good thought - I was initially banking on it creating a separate artifact for each file, but unfortunately it just doesn't work that way.

Updated this